### DISCUSSION TITLE
NEW CATEGORIES HAVE LAUNCHED: Programming Help and Machine Learning  :flying_saucer:!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#52655] [Announcements]
Houston, we’ve got a solution! We’ve noticed many of you have come to the GitHub Community with your programming and project questions as well as interest in Machine Learning and AI. Most of these posts were landing in the General category and it was easy to lose those in the hubbub of the community. So, we decided to create a new home for these posts with TWO new categories.

We have added [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) for questions about coding, project help, career advice, and more. Before you post, please review our [Programming Help category guidelines](https://github.com/orgs/community/discussions/51384) to learn more about the category.

Machine Learning projects and models are pretty awesome so it made sense to make our second category, [Machine Learning and MLOps](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops). Show off your models, and discuss the latest AI news here!

With the addition of these categories, also comes new labels to use: 

* AI or ML Model: Question about a specific model or I’ve created a new one to share
* MLOps: Trying to coordinate the machine learning process on GitHub
* Programming Help: Programming languages, open source, and software development.
* Troubleshooting: This “almost” works, there’s a detail I think I’m missing

These categories will be trialed from April 1st to September 30th, 2023 and we may be checking in with those in the community to see how they’re feeling about it, but your feedback is welcome at any time :ear:!

Happy posting and programming :minidisc:!

**Comment by anjalipatel-dev** [comment]
Amazing 😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍🤩😍🤩😍🤩😍😍🤩🤩🤩🤩🤩🤩🤩🤩

**Reply by zhengtu2323** [reply]
nice

**Comment by anjalipatel-dev** [comment]
🌺👍🌺👍🍒🥭📸👍🥭📱🌺🌺🍒📱🥭📸👍🍒🍒📱🌺🌺👍📱📱🌌🍎🍍🌺👍🍒🥭📸👍😍🍒🌺🥭📸📱🌼🌌🍓🍎🍍🍌🌌🍓⛈️🥭🍒🍒🥝🌽💧🌨️🌧️⛈️🗻⛲💒🌩️🍅🍊🥑🍉🌨️🌾🌱☁️🌩️💒🍌🍍🍎🌼📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📸📸📸🌺🌺 filling nature in ml using devices.

**Reply by mazr711001** [reply]
Focus to target

**Comment by anjalipatel-dev** [comment]
💫🌟⭐✨🌟⭐💫🌟✨⭐⭐💫🌟🌟🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌝🌜🌛🌜🌛🌛🌛🌛🌛🌛🌛🌛🌛🌛🌛🌛🌛📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱🌼🍎🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🍎📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱🌾🌼🍎🍉🌨️🌾🌼🍎🍉🌨️🌾🌼🍎🍉🌨️🌾🌼🍎🍉🌨️🌨️🌼🍉🍉🌨️🌨️🌼🍎🌨️🌨️🌼🌼🍎🍉🌨️🌼🌼🍎🍉🌨️🌼🍎🍉🌨️🌼🍎🍉🌨️🍎🍎🍉🌨️🌨️🌼🌼🍎🌨️🌼🍎🌨️🌼🍎🍉🌨️🍎🍉🌨️🍎🌨️🌼🌨️🌨️🌾🌼🍎🍉🌾🌨️🍎🍉🌨️🌼🍎🍉🌨️🌨️🌼🍎🍉🌾🌼🍎🍉🌾🌼🍉🌨️🌨️🌼🍉🌨️🌾🌼🍎🍉🍉💧💧🌼🍎🍉💧🌨️🌼🌨️🌼🍎🍎💧🌨️🍎🌨️💧🍉🍉🍉💧💧🌨️🍉🌨️🌨️💧💧💧🍉🍎🌼🌧️💧🍉🍎🌼🌨️💧💧🍉🍉🍍🍍🥑🥝🌽💧💧🥑🍒🍊🍊🍌🍍🍎🌾🥝🍒🥭🥭🥭🍅💒🍌🍍🍎🌼🌨️🥝🍊just filled with fruits and 'The Prakriti '.

**Comment by khlaifiabilel** [comment]
Great news 😍😍

**Reply by Doubledeal1** [reply]
It's all good

**Reply by RobinKumar5986** [reply]
ok

**Reply by Ademola1987** [reply]
Nice reply

**Comment by ASMAE87692** [comment]
ppppppplplpllp📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱🌼🍎🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🌼🍎🍎📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱📱🌾🌼🍎🍉🌨️🌾🌼🍎🍉🌨️🌾🌼🍎🍉🌨️🌾🌼🍎🍉🌨️🌨️🌼🍉🍉🌨️🌨️🌼🍎🌨️🌨️🌼🌼🍎🍉🌨️🌼🌼🍎🍉🌨️🌼🍎🍉🌨️🌼🍎🍉🌨️🍎🍎🍉🌨️🌨️🌼🌼🍎🌨️🌼🍎🌨️🌼🍎🍉🌨️🍎🍉🌨️🍎🌨️🌼🌨️🌨️🌾🌼🍎🍉🌾🌨️🍎🍉🌨️🌼🍎🍉🌨️🌨️🌼🍎🍉🌾🌼🍎🍉🌾🌼🍉🌨️🌨️🌼🍉🌨️🌾🌼🍎🍉🍉💧💧🌼🍎🍉💧🌨️🌼🌨️🌼🍎🍎💧🌨️🍎🌨️💧🍉🍉🍉💧💧🌨️🍉🌨️🌨️💧💧💧🍉🍎🌼🌧️💧🍉🍎🌼🌨️💧💧🍉🍉🍍🍍🥑🥝🌽💧💧🥑🍒🍊🍊🍌🍍🍎🌾🥝🍒🥭

**Comment by modxii** [comment]
Good

**Reply by modxii** [reply]
Good

**Reply by 0xIvanChang** [reply]
Good

**Comment by VaniaVania** [comment]
I hope people will become more interacted with CS at all.

**Reply by Ademola1987** [reply]
better

**Comment by mprabhashmp** [comment]
❤️

**Comment by kakar930** [comment]
NEW CATEGORIES HAVE LAUNCHED: Programming Help and Machine Learning 🛸! #52655

**Comment by Yooo-Its-King** [comment]
Nice. Help with programming for people like me!

**Comment by ASMAE87692** [comment]
On Tuesday, April 25, 2023 at 02:44:25 PM GMT+1, TALAQALOTAIBIPMP ***@***.***> wrote:  
 
 


بببببلبلبلب 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 🌼 🍎 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🌼 🍎 🍎 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 📱 🌾 🌼 🍎 🍉 🌨️ 🌾 🌼 🍎 🍉 🌨️ 🌾 🌼 🍎 🍉 🌨️ 🌾 🌼 🍎 🍉 🌨️ 🌨️ 🌼 🍉 🍉 🌨️ 🌨️ 🌼 🍎 🌨️ 🌨️ 🌼 🌼 🍎 🍉 🌨️ 🌼 🌼 🍎 🍉 🌨️ 🌼 🍎 🍉 🌨️ 🌼 🍎 🍉 🌨️ 🍎 🍎 🍉 🌨️ 🌨️ 🌼 🌼 🍎 🌨️ 🌼 🍎 🌨️ 🌼 🍎 🍉 🌨️ 🍎 🍉 🌨️ 🍎 🌨️ 🌼 🌨️ 🌨️ 🌾 🌼 🍎 🍉 🌾 🌨️ 🍎 🍉 🌨️ 🌼 🍎 🍉 🌨️ 🌨️ 🌼 🍎 🍉 🌾 🌼 🍎 🍉 🌾 🌼 🍉 🌨️ 🌨️ 🌼 🍉 🌨️ 🌾 🌼 🍎 🍉 🍉 💧 💧 🌼 🍎 🍉 💧 🌨️ 🌼 🌨️ 🌼 🍎 🍎 💧 🌨️ 🍎 🌨️ 💧 🍉 🍉 🍉 💧 💧 🌨️ 🍉 🌨️ 🌨️ 💧 💧 💧 🍉 🍎 🌼 🌧️ 💧 🍉 🍎 🌼 🌨️ 💧 💧 🍉 🍉 🍍 🍍 🥑 🥝 🌽 💧 💧 🥑 🍒 🍊 🍊 🍌 🍍 🍎 🌾 🥝 🍒 🥭

**Comment by Kitkat77** [comment]
I have boyfriend names Michael Vickers at home who has the abilities of AL who target my dispatched ears where I have been hearing him threat possible by a robot machine who control my sound amplifier maybe I don't wear hearing aid only invisibility and also burning volume and voltage on my body physical on my moto g pure imei:354919471172299 check it out thank you android OU possible pls take Jason long off my dashboard on my URL and systems thank you he's doing some kind if Spanish and possible Korea languages in some Samsung imei:355151133819841 on mikes Vickers thank you

**Comment by Kitkat77** [comment]
Pls destroyed the android OS

**Comment by RobinKumar5986** [comment]
ok

**Comment by tamaosaka** [comment]
![new.pdf](https://github.com/community/community/files/11587047/new.pdf)

**Comment by tamaosaka** [comment]
![my-document.pdf](https://github.com/community/community/files/11587048/my-document.pdf)

**Comment by tamaosaka** [comment]
![my-document (1).pdf](https://github.com/community/community/files/11587050/my-document.1.pdf)

**Reply by tamaosaka** [reply]
？
i am so sorry too late.
my work is demolition.

april, hurt a lot.

and upload file is good?

still i am not use github law.
repository, issue...?

**Comment by tamaosaka** [comment]
![a8 death my-document.pdf](https://github.com/community/community/files/11587051/a8.death.my-document.pdf)

**Comment by tamaosaka** [comment]
i big misstake to github ???? years.

so this time,
study github's law first.

and 49 years old.
smartphone button finger tap very difficult because my age and my eye is uncle eye, need uncle glass.

en. skill 10 years old level.

email. is my internet song or other.

In any case, I sincerely apologize for the delay.I recently learned that there is a translator here. But when the old finger pulls the button, it becomes a button on the side.Even now, I miss the wrong button 10 times while correcting typos.I couldn't keep up with the speed of the thread, so when I got home, I ate and took a bath and immediately fell asleep in bed.The reason is that I work as a demolition worker in the construction industry, and if I don't go to bed right away, I can't work the next day. It's hard not to be able to eat rice, so there's no need to worry about choosing one. Because it swings like a swing, and it's dangerous if you don't get enough sleep. Mortar is falling off, and nails that have rusted through the shoes on the soles of the feet are common.So, worker safety, negotiations with the main contractor, 4 hours round trip driving, 5 hours sleep a,I'm sorry I uploaded the file here and I think it's also against the rules here, but I think it's a disappointment for everyone, maybe you mean my internet song? but How to say, any programming experience is up to Hello World.I haven't been able to create variables, frames, or anything for 30 years.From what I know, I look at it every day, making buttons like this, but maybe a layer higher, etc. If I can do that for an hour a day (watching), I understand that there is a little understanding. The past few months by what I did directly to the email address at.It's a big loss.I am very sorry. We deeply apologize.About 3 days ago, even though I was communicating with a company that seemed to be an offer, I was ignorant of programming and software mechanisms.I thought I had canceled my subscription, and on the same day a claim for damages was filed for violating our privacy policy.I wanted to study at the speed of slowly climbing the stairs one by one, but I knew it was me, so I promise that if anything happens later, it won't happen to you. account What can I say, I was studying how android works because downloading one program was not easy on my smartphone.I also do bands and silver jewelry, and I don't know if the customer's deadline is my story.It's only rice today at the demolition worker.Please take a day off today 3 plans at 6 in the morning? measures? But if you copy and paste it into the first category you see, now it's angle 60, comments from those who read it? thoughts? I will do this further, I am that kind of person, nice to meet you. However, the data will be updated first.I need your permission to paste the URL over there here So when I go to that site,Even now, I can't understand clearly because of my vague level of English ability.Last night, if a lot of contacts were true,All of us really work together to create a living environment for our descendants,As far as I know, children with only bones and skin are 30 years old.I'm from Japan, but I'm not denying what kind of volunteer group activities, but why i I thought that subscription was 1,000 yen instead of 100,000 yen per month.Defend yourself a little.What I said here is my memory, and since I have not disclosed the name of the company or the technology, I am sure that you are protecting your company's information.Please make the trial directly to me. I have submitted all my personal information to your company.If something happens to other companies, we will make decisions and take action at that time.However, the Internet has been around for many years.Concerns over semiconductor shortages. Now is the moment when we have to do something more than the cost of the dispute. I can't even relax Materials that are still in short supply will appear in the future. When we see it, we use it, make it money, and start a business again.AI takes precedence. After 2,000 years of capitalist socialism, we are not worried about running out of material here and now,Materials will run out quickly.I heard that money is about to be hard to come by.It's an embarrassing story because you don't know what it's like unless you go to the dark web and dark net and experience it yourself.Now let's get back to uploading. Duplication is a waste of stamina.so far,I am sorry.I am very sorry that you have to spare time for that as well. I apologize profusely.Thank you very much for your support.Also, reading up to this point is exhausting.PS,Until now, I didn't even know if the mailing list was addressed to me.Postscript is over from now,What I can do is knowledge, experience, quality control, quality assurance or I think it's also good to think in terms of the world as a whole. Steel structure workers, medical automatic equipment assemblers, and analog circuit analysis nano-levels cost a billion yen to analyze solder chipping to the nano level (I'm not bragging, but if you replace the whole thing, it will cost a little more. with new insurance maintenance)8mm video mechanism, analog circuit, video circuit, sound circuit, their operation check maintenance (oscilloscope (analog)), and I'm sorry I didn't mention the name. One executive candidate in the aviation industry Japan.32 years old Collapsed from overwork (weight 40 kg), divorced, 49 years old17 years.Before I knew it, there were musical instruments, trash, silver, and paintings in this room.A 49-year-old middle-aged man has a habit of straying from his smartphone for a while.Someone will do it for me so far.Again, it's been a long time, sorry.After returning to the upload, I am replying to the company that I registered over there.If you have any questions, please don't hesitate to ask me anything, even if I don't understand.Well then.t can't be done, what is necessary to carry it out, we wealthy people don't know.There is even an announcement of how he himself did in some cyberattack the other day.If you put security software on this device,If you uninstall, what is your personal information handling authority? is our freedom (words, sentences may be different),But we are saying the same thing. I agree with you at the time of installation.This exchange of mine also uses that server

**Comment by tamaosaka** [comment]
i big misstake to github ???? years.

so this time,
study github's law first.

and 49 years old.
smartphone button finger tap very difficult because my age and my eye is uncle eye, need uncle glass.

en. skill 10 years old level.

email. is my internet song or other.

In any case, I sincerely apologize for the delay.I recently learned that there is a translator here. But when the old finger pulls the button, it becomes a button on the side.Even now, I miss the wrong button 10 times while correcting typos.I couldn't keep up with the speed of the thread, so when I got home, I ate and took a bath and immediately fell asleep in bed.The reason is that I work as a demolition worker in the construction industry, and if I don't go to bed right away, I can't work the next day. It's hard not to be able to eat rice, so there's no need to worry about choosing one. Because it swings like a swing, and it's dangerous if you don't get enough sleep. Mortar is falling off, and nails that have rusted through the shoes on the soles of the feet are common.So, worker safety, negotiations with the main contractor, 4 hours round trip driving, 5 hours sleep a,I'm sorry I uploaded the file here and I think it's also against the rules here, but I think it's a disappointment for everyone, maybe you mean my internet song? but How to say, any programming experience is up to Hello World.I haven't been able to create variables, frames, or anything for 30 years.From what I know, I look at it every day, making buttons like this, but maybe a layer higher, etc. If I can do that for an hour a day (watching), I understand that there is a little understanding. The past few months by what I did directly to the email address at.It's a big loss.I am very sorry. We deeply apologize.About 3 days ago, even though I was communicating with a company that seemed to be an offer, I was ignorant of programming and software mechanisms.I thought I had canceled my subscription, and on the same day a claim for damages was filed for violating our privacy policy.I wanted to study at the speed of slowly climbing the stairs one by one, but I knew it was me, so I promise that if anything happens later, it won't happen to you. account What can I say, I was studying how android works because downloading one program was not easy on my smartphone.I also do bands and silver jewelry, and I don't know if the customer's deadline is my story.It's only rice today at the demolition worker.Please take a day off today 3 plans at 6 in the morning? measures? But if you copy and paste it into the first category you see, now it's angle 60, comments from those who read it? thoughts? I will do this further, I am that kind of person, nice to meet you. However, the data will be updated first.I need your permission to paste the URL over there here So when I go to that site,Even now, I can't understand clearly because of my vague level of English ability.Last night, if a lot of contacts were true,All of us really work together to create a living environment for our descendants,As far as I know, children with only bones and skin are 30 years old.I'm from Japan, but I'm not denying what kind of volunteer group activities, but why i I thought that subscription was 1,000 yen instead of 100,000 yen per month.Defend yourself a little.What I said here is my memory, and since I have not disclosed the name of the company or the technology, I am sure that you are protecting your company's information.Please make the trial directly to me. I have submitted all my personal information to your company.If something happens to other companies, we will make decisions and take action at that time.However, the Internet has been around for many years.Concerns over semiconductor shortages. Now is the moment when we have to do something more than the cost of the dispute. I can't even relax Materials that are still in short supply will appear in the future. When we see it, we use it, make it money, and start a business again.AI takes precedence. After 2,000 years of capitalist socialism, we are not worried about running out of material here and now,Materials will run out quickly.I heard that money is about to be hard to come by.It's an embarrassing story because you don't know what it's like unless you go to the dark web and dark net and experience it yourself.Now let's get back to uploading. Duplication is a waste of stamina.so far,I am sorry.I am very sorry that you have to spare time for that as well. I apologize profusely.Thank you very much for your support.Also, reading up to this point is exhausting.PS,Until now, I didn't even know if the mailing list was addressed to me.Postscript is over from now,What I can do is knowledge, experience, quality control, quality assurance or I think it's also good to think in terms of the world as a whole. Steel structure workers, medical automatic equipment assemblers, and analog circuit analysis nano-levels cost a billion yen to analyze solder chipping to the nano level (I'm not bragging, but if you replace the whole thing, it will cost a little more. with new insurance maintenance)8mm video mechanism, analog circuit, video circuit, sound circuit, their operation check maintenance (oscilloscope (analog)), and I'm sorry I didn't mention the name. One executive candidate in the aviation industry Japan.32 years old Collapsed from overwork (weight 40 kg), divorced, 49 years old17 years.Before I knew it, there were musical instruments, trash, silver, and paintings in this room.A 49-year-old middle-aged man has a habit of straying from his smartphone for a while.Someone will do it for me so far.Again, it's been a long time, sorry.After returning to the upload, I am replying to the company that I registered over there.If you have any questions, please don't hesitate to ask me anything, even if I don't understand.Well then.t can't be done, what is necessary to carry it out, we wealthy people don't know.There is even an announcement of how he himself did in some cyberattack the other day.If you put security software on this device,If you uninstall, what is your personal information handling authority? is our freedom (words, sentences may be different),But we are saying the same thing. I agree with you at the time of installation.This exchange of mine also uses that server

**Comment by flashbtc044** [comment]
About FLASH BTC TRANSACTION (Core Network) Full Version 10.0.0

FLASH BTC TRANSACTION (Core Network) Full Version 10.0.0 is an open-source software that sends fake btc to any node (exchanger, peer-to-peer wallet, open-source, ledger wallet).

The Features of FLASH BTC TRANSACTION (Core Network) Full Version 7.0.0

- Works in all countries, no IP address restrictions
-Send transactions with confirmation
-No transaction limit per day
-No amount limit per day
-Can send up to 500 BTC per transaction
-Stay in the wallet for more than 180 days
-100% anonymous (erases your history after each transaction)
-You can change transaction fees
- you can receive the balance automatically without confirmation.
- Transferable from one wallet to another
- Supports all addresses in btc format (segwit, legacy, bch)

What you need to use this software:

-A computer with a windows system (windows 7 at least)
-It also works over RDP

Admin contact:

Telegram: @Cks_admin

**Comment by nullsecX270** [comment]
happy learning

**Comment by mike-koala-bear** [comment]
nice

**Comment by kkkkkkhx** [comment]
ccc

**Comment by Spixmacaw** [comment]
Houston, we’ve got a solution! We’ve noticed many of you have come to the GitHub Community with your programming and project questions as well as interest in Machine Learning and AI. Most of these posts were landing in the General category and it was easy to lose those in the hubbub of the community. So, we decided to create a new home for these posts with TWO new categories.

We have added [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) for questions about coding, project help, career advice, and more. Before you post, please review our [Programming Help category guidelines](https://github.com/orgs/community/discussions/51384) to learn more about the category.

Machine Learning projects and models are pretty awesome so it made sense to make our second category, [Machine Learning and MLOps](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops). Show off your models, and discuss the latest AI news here!

With the addition of these categories, also comes new labels to use:

AI or ML Model: Question about a specific model or I’ve created a new one to share
MLOps: Trying to coordinate the machine learning process on GitHub
Programming Help: Programming languages, open source, and software development.
Troubleshooting: This “almost” works, there’s a detail I think I’m missing
These categories will be trialed from April 1st to September 30th, 2023 and we may be checking in with those in the community to see how they’re feeling about it, but your feedback is welcome at any time 👂!

Happy posting and programming 💽!

**Comment by neelkal** [comment]
Good

**Comment by reeveng** [comment]
Nice! Now get the poll going on the new github homepage with which people actually want! https://github.com/orgs/community/discussions/categories/feed

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
